db/migrate/20220520105754_create_markers.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMarkers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :markers, id: :uuid do |t|
+      t.belongs_to :vacancy, null: false, foreign_key: true, type: :uuid
+      t.belongs_to :organisation, null: false, foreign_key: true, type: :uuid
+      t.st_point :geopoint, geographic: true
+      t.date :publish_on, null: false
+      t.datetime :expires_at, null: false
```

Comment:
It might also be worth to add an index on the date/datetime columns because we do a _lot_ of scoping by date in the app. Just noticed we don't have any on the main vacancies table so we might want to do that too (separately) 💡

Reply:
Correction – vacancies do have an index on `expires_at`, just not `publish_on` 🤨